client/Dockerfile

```diff
@@ -4,12 +4,28 @@ WORKDIR /usr/src/app/client
 
 RUN apk add aws-cli git
 
+RUN npm -g config set user root
 RUN npm install --global gulp-cli
 COPY --from=golang:1.16-alpine /usr/local/go/ /usr/local/go/
+
+RUN npm install --location=global @sentry/cli
+ENV SENTRYCLI_SKIP_DOWNLOAD=1
+# RUN mkdir -p /usr/src/app/client
```

Comment:
Should we keep the commented out areas? Maybe we don't need them anymore

Reply:
Thanks! Removed.